test/jira-tests.js

```diff
@@ -30,11 +30,7 @@ describe('Jira API Tests', () => {
     });
 
     it('Constructor with no auth credentials', () => {
-      const {
-        username,
-        password,
-        ...options
-      } = getOptions();
+      const { username, password, ...options } = getOptions(); // eslint-disable-line no-use-before-define, max-len
```

Comment:
Eslint or babel-eslint currently has a bug with object destructuring and the `...` operator.